platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEvent.java

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.android.telemetry.Event;
+
+import android.os.Bundle;
+import android.os.Parcel;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class PerformanceEvent extends Event {
+  private static final String PERFORMANCE_TRACE = "performance.trace";
+
+  @SerializedName("event")
+  private final String event;
+
+  @SerializedName("created")
+  private final String created;
+
+  @SerializedName("sessionId")
+  private final String sessionId;
+
+  @SerializedName("data")
```

Comment:
FYI if the field name matches the serialized variant, there is no need to declare `@SerializedName`